### ISSUE TITLE
Add Listenbrainz support.

### ISSUE THREAD

**Issue by soredake** [shymega/scrob#1]
It would be nice to have an https://listenbrainz.org/ as target option, this service is more lively than gnu-fm.

**Comment by shymega** [comment]
Hi,

soredake <notifications@github.com> writes:

> https://listenbrainz.org/

I take it this proposal would act as a `target` for RScrobble? I think
it would do.

In future bug reports, I would suggest that you supply more
information than just a short title, and a link. I need to know _what_
this would bring to the project, what's in it for me (the sole
maintainer, who does this for free), and other information. The
information you have supplied so far is quite limiting for me to work
with.

If you could reply to this issue with my suggestions as listed above,
I'll take another look.

Thank you for starring RScrobble and your interest in this project.

-- 
Best regards,
Dom Rodriguez

"Top-posting causes armed conflicts." ~~ anon, the internet

**Comment by shymega** [comment]
Thanks for the background information. As far as I can see, ListenBrainz doesn't appear to have a reference implementation ready yet, and the website _does_ state it's in infancy stage. I'll keep this issue open for now, but I'm not sure how to implement this into RScrobble yet. I'll need to develop an external lib as well.

**Comment by shymega** [comment]
On further analysis, it appears that there is a reference server, and API documentation. I evidently wasn't looking hard enough.

I will look into developing a library for this. Thank you for bringing it to my attention.

**Comment by soredake** [comment]
Thank YOU for the great scrobbler that i very need!

**Comment by shymega** [comment]
Good. That link gives me sufficient information to begin developing a library for listenbrainz.

**Comment by shymega** [comment]
It's not necessary to close this, I would rather it remain open.

**Comment by shymega** [comment]
Re-opening. Please don't close issues when it's still being worked on. I'm involved with a Listenbrainz library currently, working on reviewing and adding functionality.

**Comment by soredake** [comment]
Sorry, I don't want to keep issues for software that I don't use anymore.

**Comment by shymega** [comment]
Well, this is my issue tracker, and it's up to me. If you don't want updates, _unsubscribe_ from the issue. I need to track progress on this feature.

**Comment by soredake** [comment]
Recreate this issue yourself, please.

**Comment by shymega** [comment]
> Recreate this issue yourself, please.

I'm sorry, but you can't decide how issues work on a project you don't own. I appreciate you no longer wish to receive updates - in which case, this issue has an 'unsubscribe' button you can use.

However, I want this issue to remain open. It acts as a tracking issue. Closing it gives people the impression (from a glance) that it's no longer being worked on when it _is_.

I will re-open this issue. Please then unsubscribe from it, and you won't be notified. But please do _not_ close the issue, because that's not your decision to make - it's a feature I want too.

If you need graphics or more description on how to unsubscribe, please let me know. However, if the issue remains being opened and closed, I will have to restrict your access to the repository, because it doesn't help. That is nothing personal, it's merely because I want this issue to remain open.

I apologise for the time this feature has taken to be implemented, but I've been busy with college and other things. I'm currently working on the `refactor` branch which will eventually rework the structure.

Thank you for your interest.

**Comment by shymega** [comment]
Right, I'm gonna have to remove you from the repository, seeing as my response wasn't heeded. This issue will remain open.
